### PR TITLE
and now I forgot to use the latest version of mu...

### PR DESCRIPTION
**Pull request by dmarticus**
same energy; need to use the latest mu version to compile sbt-mu-srcgen

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -8,11 +8,15 @@ addCommandAlias("ci-test", "scalafmtCheckAll; scalafmtSbtCheck; test; publishLoc
 addCommandAlias("ci-docs", "github; mdoc; headerCreateAll")
 addCommandAlias("ci-publish", "github; ci-release")
 
+val V = new {
+  val mu          = "0.23.0"
+}
```
Comment (juanpedromoreno) [inline build.sbt:13]: **suggestion**: I'd leave it as a simple lazy val

**Comment by dmarticus** [comment]
argh I don't know why this won't build I've run the formatter twice now

File: build.sbt [hunk 2]
```diff
@@ -8,11 +8,15 @@ addCommandAlias("ci-test", "scalafmtCheckAll; scalafmtSbtCheck; test; publishLoc
 addCommandAlias("ci-docs", "github; mdoc; headerCreateAll")
 addCommandAlias("ci-publish", "github; ci-release")
 
+lazy val V = new {
+  val mu          = "0.23.0"
+}
```
Comment (juanpedromoreno) [inline build.sbt:13]:
```suggestion
lazy val muV = "0.23.0"
```

File: build.sbt [hunk 3]
```diff
@@ -8,11 +8,15 @@ addCommandAlias("ci-test", "scalafmtCheckAll; scalafmtSbtCheck; test; publishLoc
...
+
 lazy val core = project
   .settings(moduleName := "mu-srcgen-core")
   .settings(
     libraryDependencies ++= Seq(
-      "io.higherkindness"          %% "mu-rpc-service"      % "0.22.3",
+      "io.higherkindness"          %% "mu-rpc-service"      % V.mu,
```
Comment (juanpedromoreno) [inline build.sbt:19]:
```suggestion
      "io.higherkindness"          %% "mu-rpc-service"      % muV,
```

File: build.sbt [hunk 4]
```diff
@@ -34,7 +38,7 @@ lazy val plugin = project
     scriptedLaunchOpts ++= Seq(
       "-Xmx2048M",
       "-XX:ReservedCodeCacheSize=256m",
-      "-Dmu=0.22.3",
+      s"-Dmu=${V.mu}",
```
Comment (juanpedromoreno) [inline build.sbt]:
```suggestion
      s"-Dmu=$muV",
```

**Review by juanpedromoreno — APPROVED** [review]

**Pull request merged**